UnityProject/Assets/Scripts/Messages/Server/UpdateTileMessage.cs

```diff
@@ -45,6 +45,7 @@ public struct delayedData
 
 		public override void Process(NetMessage msg)
 		{
+			if (CustomNetworkManager.IsServer) return;
```

Comment:
breaks gas overlays I think, unless you took care of it

Reply:
ok, will look in to

Reply:
looked into seem to be fine, tested with build host to client build ,
noted that you can't see if other people on fire or maybe just host

Reply:
People on fire should ven unrelated. That’s part of the player’s overlays